src/services/users.js

```diff
@@ -182,5 +183,116 @@ class UsersService {
       res.boom.badImplementation();
     }
   }
+
+  async addProducts (req, res) {
+    try {
+      const user = await User.findById(req.params.id).exec();
+      if (!user) {
+        return res.boom.notFound('User not found');
+      }
+
+      const reqProduct = await Product.findById(req.body.productId).exec();
```

Comment:
Use getProducts method from productsSevice for his

Reply:
but productsService takes the request query but i use req.body for this method's parameters and productsService returns found product as response while i need only the data of found products, not a response

Reply:
So in this case add findProduct method to products service. Lets follow pattern: each service responsible for one model